src/hostname/hostnamectl.c

```diff
@@ -312,6 +313,14 @@ static int print_status_info(StatusInfo *i) {
                         return table_log_add_error(r);
         }
 
+        if (!isempty(i->ansi_color)) {
+                r = table_add_many(table,
+                                   TABLE_FIELD, "Ansi Color",
+                                   TABLE_STRING, i->ansi_color);
+                if (r < 0)
+                        return table_log_add_error(r);
+        }
```

Comment:
I think instead of adding a separate field for this, I'd just color the three hostname fields at the beginning of this table with this color.

for that use the TABLE_SET_COLOR pseudo cell type. But you need to prefix/suffix the ANSI seq intro/outro for this to work.